enterprise/models.py

```diff
@@ -361,14 +361,11 @@ def sync_learner_profile_data(self):
 
         Returns False if enterprise customer does not have any identity provider.
         """
-        try:
-            # pylint: disable=no-member
-            return (
-                self.enterprise_customer_identity_provider and
-                self.enterprise_customer_identity_provider.sync_learner_profile_data
-            )
-        except ObjectDoesNotExist:
-            return False
+        # pylint: disable=no-member
```

Comment:
Didn't we need to make any change where this property is being used? Also, dd you test this with multiple IdP's?

Reply:
I don't think we need to make any more changes because it's still returning a boolean field. Haven't been able to test this locally as I couldn't configure multiple IDPs locally but the relevant test cases didn't break and I'm planning to test it on stage once the feature is deployed there.